internal/stackdiag.go

```diff
@@ -420,7 +422,13 @@ func scheduleJobs(k *Kubectl, ns string, recordErr func(error), state *diagJobSt
 			return nil
 		}
 
-		recordErr(state.scheduleJob(typ, esName, resourceName, isTLS))
+		esSecretName, esSecretKey, err := determineESCredentialsSecret(k, ns, esName)
+		if err != nil {
+			// If no credentials secret was found this is a fatal error.
+			panic(fmt.Errorf("while determining Elasticsearch credentials secret: %w", err))
```

Comment:
```suggestion
			recordErr(fmt.Errorf("while determining Elasticsearch credentials secret: %w", err))
			return
```
to try other clusters/resources?

Reply:
I went back and forth on this, and initially had it this way, but chose panic instead as it seemed a pretty bad situation, but I could see the case where we are still getting relevant data and should attempt to continue. I've adjusted this.